orttraining/orttraining/training_api/training_session.cc

```diff
@@ -112,7 +112,16 @@ Status TrainingSession::CopyParametersToBuffer(OrtValue& parameters_buffer, cons
 }
 
 Status TrainingSession::CopyBufferToParameters(OrtValue& parameters_buffer, const bool trainable_only) {
-  return module_->CopyBufferToParameters(parameters_buffer, trainable_only);
+  const bool was_nominal_state = state_->module_checkpoint_state.is_nominal_state;
```

Comment:
nit: can be moved below closer to its usage.

Reply:
`Module::CopyBufferToParameters` updates the checkpoint `is_nomial_checkpoint` flag to `false` at the end of the function.

Reading the flag later will always yield `false`. Which is why I read the flag first.